test/integration/suites/get-authorized-entries/04-create_registration

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+SIZE=10
+
+# Create entries for uid 1001
+for ((m=1;m<=$SIZE;m++)); do
+  log-debug "creating registration entry: $m"
+  docker compose exec -T spire-server \
+    /opt/spire/bin/spire-server entry create \
+    -parentID "spiffe://domain.test/spire/agent/x509pop/$(fingerprint conf/agent/agent.crt.pem)" \
+    -spiffeID "spiffe://domain.test/workload-$m" \
+    -selector "unix:uid:1001" \
+    -ttl 0 &
+done
+
+for ((m=1;m<=$SIZE;m++)); do
+  check-synced-entry "spire-agent" "spiffe://domain.test/workload-$m"
+done
```

Comment:
let's simplify CI and make each step distinct; checking in agent that entries are synced is separate from their creation